app/models/pull_request.rb

```diff
@@ -19,17 +19,28 @@ class PullRequest < ApplicationRecord
 
   def self.update_with_github(gh_pull_request)
     PullRequest.where(github_id: gh_pull_request['id']).first_or_initialize.tap do |pull_request|
+      # get current status. GitHub API does not expose it as an atttribute of a PR
+      # However, https://github.com/search does
+      repo_id = gh_pull_request['base']['repo']['id']
+      status = begin
+                statuses = Github.client.combined_status(repo_id, gh_pull_request[:head][:sha])
+                statuses[:state]
+               rescue StandardError => e
+                 Raven.capture_message('validate status', extra: { error: e.inspect, github_data: gh_pull_request })
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long. [116/100]](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5456863)